app/src/main/java/com/sdpteam/connectout/chat/ChatModel.java

```diff
@@ -0,0 +1,92 @@
+package com.sdpteam.connectout.chat;
+
+import static com.sdpteam.connectout.profile.EditProfileActivity.NULL_USER;
+
+import android.view.View;
+import android.widget.ListAdapter;
+
+import androidx.annotation.NonNull;
+
+import com.firebase.ui.database.FirebaseListAdapter;
+import com.firebase.ui.database.FirebaseListOptions;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.Query;
+
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+public class ChatModel implements ChatDirectory {
+
+    private final DatabaseReference firebaseRef;
+    public final String CHATS_PATH_STRING = "Chats";
+
```

Comment:
why use public scope for these two? They are only use in this class

Reply:
thats a very good point. Im changing it now